fossunited/payments/doctype/razorpay_payment/test_razorpay_payment.py

```diff
@@ -0,0 +1,137 @@
+import json
+
+import frappe
+from faker import Faker
+from frappe.tests.utils import FrappeTestCase
+
+from fossunited.doctype_ids import EVENT, RAZORPAY_PAYMENT
+
+fake = Faker()
+
+
+class TestRazorpayPayment(FrappeTestCase):
+    def setUp(self):
+        self.admin_user = frappe.get_doc("User", "Administrator")
+        self.normal_user = frappe.get_doc("User", "test1@example.com")
+
+    def tearDown(self):
+        frappe.set_user("Administrator")
+
+        # Delete any created Razorpay Payment documents
+        payments = frappe.get_all(RAZORPAY_PAYMENT, filters={"document_name": self.event})
+        for payment in payments:
+            frappe.delete_doc(RAZORPAY_PAYMENT, payment.name)
+
+        # Delete created event
+        frappe.delete_doc(EVENT, self.event)
+
+        frappe.db.commit()
+
+    @staticmethod
+    def generate_random_permalink(max_length=20):
+        words = fake.words(nb=3)
+        permalink = "_".join(word.lower() for word in words)
+
+        return permalink[:max_length]
+
+    def create_dummy_event(self):
```

Comment:
Why not just create event in `setUp` and assign it as `self.event`? The `tearDown` will make more sense in that way(reference to above comment by Rahul)